src/combinatorics/quiver.lean

```diff
@@ -23,7 +23,7 @@ structure quiver (V : Type u) :=
 
 /-- A wide subquiver `H` of `G` picks out a set `H a b` of arrows from `a` to `b`
     for every pair of vertices `a b`. -/
-def wide_subquiver {V} (G : quiver V) :=
+def wide_subquiver {V} (G : quiver.{v+1} V) :=
```

Comment:
This would be exceeding the standard elsewhere in mathlib, but perhaps a comment saying why the universe annotation is needed here?